src/main/java/org/myrobotlab/service/interfaces/AxisPublisher.java

```diff
@@ -0,0 +1,9 @@
+package org.myrobotlab.service.interfaces;
+
+import org.myrobotlab.service.data.AxisData;
+
+public interface AxisPublisher {
```

Comment:
this should have an attachAxisListener in this interface.

Reply:
fixed